kalitectl.py

```diff
@@ -372,6 +373,14 @@ def start(debug=False, args=[], skip_job_scheduler=False):
     # TODO: What does not the production=true actually do and how can we
     # control the log level and which log files to write to
 
+    # Deal with command-line options that affect the django settings module
```

Comment:
Adding it to the docopt `[options]` would be a good solution IMO since both `status` and `stop` could make use of it (in addition to reading the port from a .pid file as suggested).